### ISSUE TITLE
get Fstype is empty on windows

### ISSUE THREAD

**Issue by rustbomber** [shirou/gopsutil#1029]
**Describe the bug**
[A clear and concise description of what the bug is.]

**To Reproduce**
```go
disk_stat, _ := disk.Usage(disk_partition.Mountpoint)
println(disk_stat.String())
```

is output:

```son
{"path":"C:","fstype":"","total":136041693184,"free":11213991936,"used":124827701248,"usedPercent":91.7569447472013,"inodesTotal":0,"inodesUsed":0,"inodesFree":0,"inodesUsedPercent":0}
```

**Expected behavior**
return Fstype.

**Environment (please complete the following information):**
 - [x] Windows: [windows 10 / windows 2019]

**Additional context**
complie on windows 10

**Comment by shirou** [comment]
Yes, Fstype on Windows is [not implemented yet.](https://github.com/shirou/gopsutil/blob/master/v3/disk/disk_windows.go#L58) PR is always welcome!

**Comment by rasa** [comment]
It appears this was fixed in https://github.com/shirou/gopsutil/pull/1407 , so this ticket can be closed.
